frontend/www/js/omegaup/components/arena/ContestPractice.vue

```diff
@@ -154,29 +149,26 @@ export default class ArenaContestPractice extends Vue {
   @Prop() activeTab!: string;
   @Prop({ default: null }) guid!: null | string;
   @Prop({ default: null }) problemAlias!: null | string;
+  @Prop({ default: () => [] }) runs!: types.Run[];
 
   T = T;
   ui = ui;
   currentClarifications = this.clarifications;
   ContestClarificationType = ContestClarificationType;
-  activeProblem: ActiveProblem | null = this.problem;
+  activeProblem: types.NavbarProblemsetProblem | null = this.problem;
   shouldShowRunDetails = false;
 
   get activeProblemAlias(): null | string {
-    return this.activeProblem?.problem.alias ?? null;
-  }
-
-  get problemRuns(): types.Run[] {
-    return this.problem?.runs ?? [];
+    return this.activeProblem?.alias ?? null;
   }
 
-  onNavigateToProblem(request: ActiveProblem) {
-    this.activeProblem = request;
-    this.$emit('navigate-to-problem', request);
+  onNavigateToProblem(problem: types.NavbarProblemsetProblem) {
+    this.activeProblem = problem;
+    this.$emit('navigate-to-problem', { problem });
   }
 
   onRunSubmitted(run: { code: string; language: string }): void {
-    this.$emit('submit-run', Object.assign({}, run, this.problem));
+    this.$emit('submit-run', { ...run, ...{ problem: this.activeProblem } });
```

Comment:
```suggestion
    this.$emit('submit-run', { ...run, problem: this.activeProblem });
```

el [spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) únicamente se usa con variables, no literales.